docs/developer-guide/docker.md

```diff
@@ -0,0 +1,92 @@
+2. Execute `docker-compose pull` - pulls the most recent version of all images
+3. Execute `docker-compose up -d` - starts all services in the background
+
+To check if everything is running correctly use the command `docker ps -a` for a list of all running containers and their corresponding containerIds.\
+The logs of each individual container can be viewed by executing `docker logs <containerId>`.
+
+To stop every service simply execute `docker-compose down`.
+
+
+## Running only selected services
+Sometimes only a subset of the provided images is needed to save computing resources or for development purposes. E.g. the database is running inside the container but IntelliJ is executing qc-atlas.\
+This is achieved by following the steps above, but instead of executing the command in step 3, use the command\
+`docker-compose up -d <serviceName1> <serviceName2> ... <serviceNameN>`\
+where each `<serviceNameX>` is replaced by the desired service names.\
+E.g. `docker-compose up -d db qc-atlas` executes the database service and the `qc-atlas` backend. However, keep in mind that some services heavily depend on each other and may not work properly if dependent services are not running.
+
+
+## Import Example Data
+Data from the [pattern-atlas-content](https://github.com/PatternAtlas/pattern-atlas-content) repository will always be imported automatically.\
+However, to import example data from the [qc-atlas-content](https://github.com/UST-QuAntiL/planqk-atlas-content) and [nisq-analyzer-content](https://github.com/UST-QuAntiL/nisq-analyzer-content) repositories make sure to follow these steps:
+
+1. Rename the `docker-compose.override.yml` file inside the [quantil-docker repository](https://github.com/UST-QuAntiL/quantil-docker) to `docker-compose.override.yml`.
+2. Provide a ssh private key file with correct access rights for the [qc-atlas-content](https://github.com/UST-QuAntiL/planqk-atlas-content) repository. The file has to be called `ssh_secret` and is located in the root folder of the [quantil-docker repository](https://github.com/UST-QuAntiL/quantil-docker).
+3. It is required that the data directory is empty before starting the container to prevent the deletion of data, see [postgres-docker documentation](https://github.com/docker-library/docs/tree/master/postgres#initialization-scripts).
+
+
+## FAQ
+
+##### How do I make sure that all necessary ports are free?
+Windows: `netstat -anp | find ":<portNumber>"` - shows processes listening on <portNumber>\
+Linux: `sudo lsof -i -P -n | grep LISTEN` - shows all processes listening any port\
+OSX: `netstat -anp tcp | grep LISTEN` for tcp and `netstat -anp udp | grep LISTEN for udp`\
+
+##### Why is the example data not imported?
+Make sure you followed the instructions above. If you still experience problems follow these steps:
+
+1. Execute `docker-compose rm -f` - **this removes all containers and volumes**
```

Comment:
This is not correct. `docker-compose rm -f` will only remove the containers, not the volumes. To delete them `docker-compose rm -vf` must be used. See: 

```bash
$ docker-compose rm --help
Removes stopped service containers.

By default, anonymous volumes attached to containers will not be removed. You
can override this with `-v`. To list all volumes, use `docker volume ls`.

Any data which is not in a volume will be lost.

Usage: rm [options] [--] [SERVICE...]

Options:
    -f, --force   Don't ask to confirm removal
    -s, --stop    Stop the containers, if required, before removing
    -v            Remove any anonymous volumes attached to containers
    -a, --all     Deprecated - no effect.
```

Reply:
Thanks, I'll add this!